libs/geoservices/api/src/lib/components/response-viewer/response-viewer.component.spec.ts

```diff
@@ -1,25 +1,18 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { async, TestBed, inject } from '@angular/core/testing';
 
 import { ResponseViewerComponent } from './response-viewer.component';
+import { HighlightPlusModule } from 'ngx-highlightjs/plus/';
+import { CommonModule } from '@angular/common';
 
 describe('ResponseViewerComponent', () => {
-  let component: ResponseViewerComponent;
-  let fixture: ComponentFixture<ResponseViewerComponent>;
-
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ ResponseViewerComponent ]
-    })
-    .compileComponents();
+      imports: [HighlightPlusModule, CommonModule],
+      providers: [ResponseViewerComponent]
+    }).compileComponents();
   }));
 
-  beforeEach(() => {
-    fixture = TestBed.createComponent(ResponseViewerComponent);
-    component = fixture.componentInstance;
-    fixture.detectChanges();
-  });
-
-  it('should create', () => {
+  it('should create', inject([ResponseViewerComponent], (component: ResponseViewerComponent<{}, {}>) => {
```

Comment:
Why inject vs getting component instance from fixture?

```js
  beforeEach(() => {
    fixture = TestBed.createComponent(ComponentToken);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });
```

I'm not saying this implementation is wrong, I just want to understand why.